server/coordinator/interface.go

```diff
@@ -0,0 +1,14 @@
+package coordinator
+
+import (
+	"github.com/privacylab/talek/common"
+)
+
+// Interface is the interface to the central coordinator
+type Interface interface {
```

Comment:
Good point. I don't think it should be in common, because the client never needs to see these internal interfaces. I created a new directory 'protocol' for these protocol/interface declarations